rpc/sendRawTransaction.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 github.com/gagliardetto
+// This file has been modified by github.com/gagliardetto
+//
+// Copyright 2020 dfuse Platform Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package rpc
+
+import (
+	"context"
+	"encoding/base64"
+
+	"github.com/gagliardetto/solana-go"
+)
+
+// SendRawTransaction submits a signed transaction to the cluster for processing.
+// The only difference between this function and SignTransaction is that the latter takes a *solana.Transaction value, as the former takes a transaction in wire format as a byte array
+func (cl *Client) SendRawTransaction(
```

Comment:
For completion, can you also please add a `SendRawTransactionWithOpts` func?

Reply:
Yes! Added :)